routes/api/tests.js

```diff
@@ -33,6 +33,16 @@ function GenerateTest(Questions, Rules) {
   return res;
 }
 
+function DuplicateQuestion(ques) {
+  var read_vals = Object.create(null);
+  for (var i = 0; i < ques.length; i++) {
+      var value = ques[i];
+      if (value in read_vals) {
```

Comment:
read_vals is always null.

Reply:
I have changed the function, please review.
Thanks